File: Logic Building - University Program/Practice Problems/Level 1/Digits and Numbers/Program 07.py
#Program 7: Reverse a 3-Digit Number

number = int(input("Enter a three digit number: "))

last_digit = number % 10 #extract last digit
middle_digit = number // 10 #removes last digit
middle_digit = middle_digit % 10 #extract middle digit
first_digit = number // 100 #extract first digit

reverse = last_digit * 100 + middle_digit * 10 + first_digit

print("Original number:", number)
print("Reversed number:", reverse)

File: Level-1_Loops/Program07.cpp
#include <iostream>
using namespace std;

int main()
{
  int N ;
  cout << "Enter a Number: " ;
  cin >> N ;
  cout << "First 10 Even Numbers : " ;
  for (int i =1; i <=N; i++) {
    if (i % 2!= 0)
    cout << i << endl;
}

    return 0;
}
